actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/tac/UserTnCActor.java

```diff
@@ -54,6 +55,7 @@ private void acceptTNC(Request request) {
     String acceptedTnC = (String) request.getRequest().get(JsonKey.VERSION);
     Map<String, Object> userMap = new HashMap();
     String userId = (String) request.getContext().get(JsonKey.REQUESTED_BY);
+    String orgId = (String) request.getRequest().get(JsonKey.ORGANISATION_ID);
```

Comment:
organisation id need not be passed in request, use the user root org id itself

Reply:
as discussed updated orgAdminTnc as hardcoded value to determine the orgAdmin tnc request

Reply:
This is needed to check the org_admin role of the user for the 'orgAdminTnc' tnctype